index.qmd

```diff
@@ -4,85 +4,81 @@ title: "Pathways to Open Science"
 
 ## Welcome
 
-**Pathways to Open Science** is a remote event series for Black environmental & marine researchers to build community for the future of data intensive science. This is an ongoing collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)), HBCU marine science professors, and now [PREreview](https://prereview.org/) and Antoinette Foster, PhD. The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams. The 2024 series ran from **January 25 to February 29.**
+**Pathways to Open Science** is a remote event series for Black environmental & marine researchers to build community for the future of data intensive science. We empower our community by sharing our stories, learning together, and building skills to expand how data science can improve, accelerate, and support our science. The 2025 series will run weekly from January 23 through Feb 27. [**Registration**](https://zoom.us/meeting/register/tJ0vc-2grzgrH9URmxCRyV34ZwjvcwCU0ciH) **is now open!**
 
 ![](images/pathways-recap-zoomie-wordcloud.png){fig-alt="2 zoom selfies of 48 Black people smiling and waving, a cartoon of a Black woman sitting, smiling, reading a book called Open Science, and a word cloud with community, personal stories, resources, as largest font words" fig-align="center" width="85%"}
 
 ## Program Details
 
 This remote event series will convene Black environmental and marine researcher community members with the goal of developing a mindset for Open Data Science: the tools and practices enabling reproducible, transparent, and inclusive practices for data intensive science. If you're feeling bogged down by folders full of files with names like "analysis_final_v5.xlsx" and/or like empowering others, we welcome you to join us!
 
-**This 6-part remote event series runs on Thursdays, January 25 - February 29**. Event format will alternate weekly (all virtual):
+This is an ongoing Openscapes collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)) and HBCU marine science professors. The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams, MSc.
```

Comment:
```suggestion
This is an ongoing Openscapes collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)) and marine science professors from 7 Historically Black Colleges and Universities (HBCUs). The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams, MSc.
```

Reply:
added in my local branch; couldn't auto-commit